nodejs/packages/layer/src/wrapper.ts

```diff
@@ -0,0 +1,22 @@
+import { NodeTracerProvider } from '@opentelemetry/node';
+import { ConsoleSpanExporter, SimpleSpanProcessor } from '@opentelemetry/tracing';
+import { AwsLambdaInstrumentation } from '@opentelemetry/instrumentation-awslambda';
+import { registerInstrumentations } from '@opentelemetry/instrumentation';
+import { AwsInstrumentation } from 'opentelemetry-instrumentation-aws-sdk';
+
+console.log('Registering OpenTelemetry');
+
+const provider = new NodeTracerProvider();
+provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+
+registerInstrumentations({
+  instrumentations: [
+    new AwsInstrumentation({
```

Comment:
only awsInstrumentation and awsLambdaInstrumentation? Can we enable all instrumentations here?